Finetuning/Finetuning_Guide.ipynb

```diff
@@ -0,0 +1,1051 @@
+      "  \"message\": \"Checks passed\",\n",
+      "  \"found\": true,\n",
+      "  \"file_size\": 23777505,\n",
+      "  \"utf8\": true,\n",
+      "  \"line_type\": true,\n",
+      "  \"text_field\": true,\n",
+      "  \"key_value\": true,\n",
+      "  \"has_min_samples\": true,\n",
+      "  \"num_samples\": 7199,\n",
+      "  \"load_json\": true,\n",
+      "  \"filetype\": \"jsonl\"\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "## We're going to check to see that the file is in the right format before we finetune\n",
+    "from together.utils import check_file\n",
+    "\n",
+    "sft_report = check_file(\"coqa_prepared_train.jsonl\")\n",
+    "print(json.dumps(sft_report, indent=2))\n",
+    "\n",
+    "assert sft_report[\"is_check_passed\"] == True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Uploading file coqa_prepared_train.jsonl: 100%|██████████| 23.8M/23.8M [00:01<00:00, 18.2MB/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Train file response: file-19c6ef51-b734-4f3c-bc17-62fbad2bd0d0\n"
+     ]
+    }
+   ],
+   "source": [
+    "## Upload the data to Together\n",
+    "\n",
+    "train_file_resp = client.files.upload(\"coqa_prepared_train.jsonl\", check=True)\n",
+    "print(f\"Train file response: {train_file_resp.id}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Launch Fine-tuning Job\n",
+    "---\n",
+    "With our data uploaded, we can now launch the finetuning job using `together.Finetune.create()`.\n",
+    "\n",
+    "Key parameters:\n",
+    "- `model`: The base model you want to finetune (e.g., `'togethercomputer/llama-2-7b-chat'`). Choose from the models available for finetuning on Together AI.\n",
+    "- `training_file`: The ID of your uploaded training JSONL file.\n",
+    "- `validation_file`: The ID of your uploaded validation JSONL file (optional, but highly recommended for monitoring).\n",
+    "- `suffix`: A custom string added to the base model name to create your unique finetuned model name (e.g., `my-coqa-ft`). Keep it short and descriptive.\n",
+    "- `n_epochs`: The number of times the model will see the entire training dataset.\n",
+    "- `n_checkpoints`: Number of checkpoints to save during training (useful for resuming or selecting the best model). Set to 1 if you only need the final model.\n",
+    "- `learning_rate`: Controls how much the model weights are updated during training. Needs tuning.\n",
+    "- `batch_size`: Number of training examples processed in one iteration. Depends on model size and available resources.\n",
+    "\n",
+    "🔗 For an exhaustive list of all the available fionetuning parameters refer to the [Together AI Fine-tuning API Reference](https://docs.together.ai/reference/post_fine-tunes) docs.\n",
```

Comment:
Looks like a typo, can you make a pass over all of the notebooks for proofreading, please?